package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
USing npm for this?  Why not foundry?

Reply:
I was hard coding the interfaces the way you showed me with [yvarb](https://github.com/ThomasdenH/yvarb/blob/master/contracts/YieldLever.sol) but Alberto [suggested](https://github.com/yieldprotocol/backd-integration/commit/fe96b71006b90c0e2995545d223b8d1afcaae03c) I import them with npm instead. I'm fine using Foundry instead. Whatever we want.

Reply:
i wasn't suggesting u hardcode those addresses, i was suggesting u look at that repo for ideas on how to work with the ladle and cauldron. sorry for the miscommunication.

per @alcueca we want to use Foundry going forward so you can go ahead and update this repo